shared/pubsub.js

```diff
@@ -359,15 +370,15 @@ const defaultMessageHandlers = {
 
   // PUB can be used to send ephemeral messages outside of any contract log.
   [NOTIFICATION_TYPE.PUB] (msg) {
-    console.debug(`[pubsub] Ignoring ${msg.type} message:`, msg.data)
+    console.warn(`[pubsub] Ignoring ${msg.type} message:`, msg.data)
   },
 
   [NOTIFICATION_TYPE.SUB] (msg) {
-    console.debug(`[pubsub] Ignoring ${msg.type} message:`, msg.data)
+    console.warn(`[pubsub] Ignoring ${msg.type} message:`, msg.data)
   },
 
   [NOTIFICATION_TYPE.UNSUB] (msg) {
-    console.debug(`[pubsub] Ignoring ${msg.type} message:`, msg.data)
+    console.warn(`[pubsub] Ignoring ${msg.type} message:`, msg.data)
```

Comment:
Do this happen often in our current usage?

Reply:
AFAIK, never, but it happened during development of the fix.